examples/tensorflow/object_detection/README.md

```diff
@@ -158,3 +158,4 @@ To export a model to the OpenVINO IR and run it using the Intel® Deep Learning
 | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: |
 |RetinaNet|None|COCO2017|33.44|194.1 (100%)|60.8 (100%)|[retinanet_coco.json](configs/retinanet_coco.json)|[Link](https://storage.openvinotoolkit.org/repositories/nncf/tensorflow/models/develop/retinanet.tar.gz)|
 |RetinaNet|Filter Pruning 40%|COCO2017|32.7 (-0.74)|107.7 (55.49%)|34.7 (57.07%)|[retinanet_coco_pruning.json](configs/pruning/retinanet_coco_pruning.json)|[Link](https://storage.openvinotoolkit.org/repositories/nncf/tensorflow/models/develop/retinanet_pruning_40.tar.gz)|
+|RetinaNet|Filter Pruning 40% + INT8 w:sym,per-tensor a:sym,per-tensor|COCO2017|32.82 (-0.62)|107.6 (55.44%)|34.7 (57.07%)|[retinanet_coco_pruning_int8.json](configs/pruning_quantization/retinanet_coco_pruning_int8.json)|[Link](https://storage.openvinotoolkit.org/repositories/nncf/tensorflow/models/develop/retinanet_int8_w_sym_t_half_a_sym_t_pruning_40.tar.gz)|
```

Comment:
```suggestion
|RetinaNet|Filter Pruning 40% + INT8 (per-tensor, symmetric for weights, asymmetric for activations)|COCO2017|32.82 (-0.62)|107.6 (55.44%)|34.7 (57.07%)|[retinanet_coco_pruning_int8.json](configs/pruning_quantization/retinanet_coco_pruning_int8.json)|[Link](https://storage.openvinotoolkit.org/repositories/nncf/tensorflow/models/develop/retinanet_int8_w_sym_t_half_a_sym_t_pruning_40.tar.gz)|
```

Reply:
Thanks for your comment. I guess this is a very long name and we will see a horizontal scroll bar for each table. Anyway, It does not relative to this PR. Your suggestion is a general improvement. I suggest adding a legend to the table to resolve your comment (AR: @evgeniya-egupova).  Also, I recommend adding your proposal to ticket #55863.

Reply:
Github rendering, which is the most likely way the user will see this table through, wraps intra-cell content automatically to fit screen width and zoom level:
https://github.com/openvinotoolkit/nncf#classification

So long lines are not an issue here, the length is basically converted to height. I suppose every MD renderer does this.

The comment relates to this PR since this PR is concerned with presenting results in a user-facing README. Here, I suggest the results to be presented in a presentable fashion.

Reply:
In this table I want to show that the quantization scheme used was the same as in int8 result above in case one wants to compare them. This notation is used in all nncf tf results, so if we want to change it, we should do it in a separate pr.